Geospatial.md

```diff
@@ -162,3 +195,19 @@ The axis order of the coordinates in WKB and bounding box stored in Parquet
 follows the de facto standard for axis order in WKB and is therefore always
 (x, y) where x is easting or longitude and y is northing or latitude. This
 ordering explicitly overrides the axis order as specified in the CRS.
+
+# Invalid geospatial values
+
+An invalid geospatial value refers to the coordinate values of a non-`null` 
+geospatial instance that are encoded in a valid WKB format, but are not 
+considered valid values under this specification. While different WKB 
+readers may interpret such values differently, the resulting output should 
+be treated as invalid.
+
+* `NaN`: Not a Number. For example, `POINT EMPTY` in WKB is represented by a 
+  `Point` with each ordinate value set to an IEEE-754 quiet NaN value.
+* `Empty geometries`: Geometries explicitly marked as empty in WKB using 
```

Comment:
I suppose that `LINESTRING EMPTY` or `POLYGON EMPTY` are WKT? Do we have canonical WKB values to demonstrate?

Reply:
Thanks for catching this. I removed the WKT representation and described the situation in plain English to avoid introducing another standard. I also added WKB examples for clarity.

Reply:
done